gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/task/COPCMessageHandlerTest.java

```diff
@@ -0,0 +1,636 @@
+package uk.nhs.adaptors.pss.translator.task;
+
+import static java.util.UUID.randomUUID;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import static uk.nhs.adaptors.common.util.FileUtil.readResourceAsString;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import javax.xml.bind.ValidationException;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.w3c.dom.Document;
+import org.xml.sax.SAXException;
+
+import lombok.SneakyThrows;
+import uk.nhs.adaptors.connector.model.PatientAttachmentLog;
+import uk.nhs.adaptors.connector.service.PatientAttachmentLogService;
+import uk.nhs.adaptors.pss.translator.exception.AttachmentLogException;
+import uk.nhs.adaptors.pss.translator.exception.InlineAttachmentProcessingException;
+import uk.nhs.adaptors.pss.translator.mhs.model.InboundMessage;
+import uk.nhs.adaptors.pss.translator.service.AttachmentHandlerService;
+import uk.nhs.adaptors.pss.translator.service.XPathService;
+
+@ExtendWith(MockitoExtension.class)
+public class COPCMessageHandlerTest {
+
+    private static final String NHS_NUMBER = "123456";
+    private static final String CONVERSATION_ID = randomUUID().toString();
+
+
+    @Mock
+    private PatientAttachmentLogService patientAttachmentLogService;
+
+    @Mock
+    private AttachmentHandlerService attachmentHandlerService;
+
+    @Mock
+    private XPathService xPathService;
+
+    @Mock
+    private Document ebXmlDocument;
+
+    @InjectMocks
+    private COPCMessageHandler copcMessageHandler;
+
+    @Test
+    public void whenHappyPathThrowNoErrors()
```

Comment:
I think that most of the tests have been named `When_HappyPath_Expect_NoErrorsThrown`- we should probably be consistent one way or another going forwards?

Reply:
Check style throws an error when using snake case as asked for. Is the error being suppressed for the formatting in EhrExtractMessageHandlerTest to pass ?